ClientSide(FrontEnd)/clay_/src/components/login.css

```diff
@@ -0,0 +1,79 @@
+.starting123{
```

Comment:
use proper naming standerd for the giving className name

Reply:
i replaced with other classname